Makefile

```diff
@@ -487,3 +497,30 @@ define list-running-pods
 	@echo "\n$(YELLOW)Local cluster $(CYAN)$1 $(NC)"
 	kubectl get pods -A --context=k3d-$1
 endef
+
+define deploy-prometheus
+	@echo "\n$(YELLOW)Local cluster $(CYAN)$1$(NC)"
+	kubectl config use-context k3d-$1
```

Comment:
Consider specifying the context inline with `helm` and `kubectl` command options instead of setting it permanently with `kubectl config`. This might be good to avoid errors when dealing with multiple clusters. `dns-tools` target is a good example for kubectl, and helm has `--kube-context` option.

Reply:
@somaritane , thx for review,  amended